### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by lenguyenthedat**
- Clean up unused imports
- Update README (to assume all commands running within source-code folder)
- Netflix folders to be created automatically by the script

### CODE REVIEW

**Comment by okuchaiev** [comment]
Thanks for your PR! However, I had to undo the commit: "Remove unused imports" because it wasn't working with it (unit tests failed). Were you using Python 3.6 ?

**Comment by lenguyenthedat** [comment]
oops, sorry, I was not using python 3 on the machine I used to create this PR 😬 
Will be careful next time, it could be due to the imports in init files. Let me try again 😄
